.github/workflows/push-to-prod-draft.yml

```diff
@@ -54,6 +64,13 @@ jobs:
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN_2}}
           balena_command: "deploy nebraltd/helium-${{ matrix.sbc }}-${{ matrix.frequency }} --logs --debug --nocache --build --draft"
+      - name: Balena Deploy (Pisces P100)
+        if: matrix.sbc == 'pisces' && success()
+        uses: nebraltd/balena-cli-action@v13.3.0
+        with:
+          balena_api_token: ${{secrets.BALENA_API_TOKEN_3}}
+          balena_command: "deploy nebraltd/helium-${{ matrix.sbc }} --logs --debug --nocache --build --draft"
```

Comment:
Why aren't we using diff fleets for diff frequencies?

Reply:
@marvinmarnold but if it's not location asserted the lora concentrator won't work anyway so it's pointless to have the frequency in the fleet (and makes it harder to use open fleets)

Agree we should remove for RAK as well.

Reply:
done :-)

Reply:
One thing is bothering me. The diagnostics container doesn't create reports if the LoRa section is not functional. So it creates confusion like the device unreachable. We need to fix that ASAP to drop freq altogether.

Reply:
I mean these:

https://github.com/NebraLtd/hm-diag/issues/288
https://github.com/NebraLtd/hm-diag/issues/289